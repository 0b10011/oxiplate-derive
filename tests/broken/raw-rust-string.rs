use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = r##"{% if "foo" == 3 %}"##]
struct Data {}

fn main() {
    let data = Data {};

    print!("{}", data);
}
