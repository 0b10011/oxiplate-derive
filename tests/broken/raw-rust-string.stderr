error: Backtrace:
       [line 4, column 36] Expected '-', found '== 3 %}'
       [line 4, column 36] Alt in "== 3 %}"
 --> tests/broken/raw-rust-string.rs:4:37
  |
4 | #[oxiplate_inline = r##"{% if "foo" == 3 %}"##]
  |                                     ^^^^^^^
